src/domUpdates.js

```diff
@@ -267,7 +267,7 @@ const displayDestinations = (allDestinations) => {
            </div>
         </div>
         <div class="lets-go-button-container" id='${location.id}'>
-            <button id="lets-go-btn" class="lets-go-btn">LETS GO</button>
+            <button id="lets-go-btn ${location.id}" class="lets-go-btn">LETS GO</button>
```

Comment:
This will actually still cause that a11y error, as you are creating two ids now. Join these IDs together with a `-`!

Reply:
Ohh that makes sense, thank you so much!